napalm_ios/ios.py

```diff
@@ -486,15 +486,13 @@ def get_lldp_neighbors_detail(self, interface=''):
 
             local_port = interface
             port_id = re.findall(r"Port id: (.+)", output)
-            port_description = re.findall(r"Port Description: (.+)", output)
+            port_description = re.findall(r"Port Description(?:\s\-|:) (.+)", output)
             chassis_id = re.findall(r"Chassis id: (.+)", output)
             system_name = re.findall(r"System Name: (.+)", output)
             system_description = re.findall(r"System Description: \n(.+)", output)
             system_capabilities = re.findall(r"System Capabilities: (.+)", output)
             enabled_capabilities = re.findall(r"Enabled Capabilities: (.+)", output)
-            remote_address = re.findall(r"Management Addresses:\n    IP: (.+)", output)
-            if not remote_address:
-                remote_address = re.findall(r"Management Addresses:\n    Other: (.+)", output)
+            remote_address = re.findall(r"Management Addresses:\n    (?:IP:|Other:) (.+)", output)
```

Comment:
I agree, but would defer to @ktbyers as it looks like he originally had created this function.